config/locales/en.yml

```
@@ -496,7 +496,7 @@ en:
       no_polygons: This search didn't use any polygons.
     international_teacher_advice:
         link: help you understand your next steps
-        text_html: If you're interested in teaching or training to teach in England as an international citizen, we can %{link}.
+        text_html: If you are interested in teaching or training to teach in England as an international citizen, we can %{link}.
```

```suggestion
        text_html: If you are interested in teaching or training to teach in England as an overseas citizen, we can %{link}.
```
How about this instead?